src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -232,6 +232,21 @@
   <data name="Globals_LaunchModeMaximized.Content" xml:space="preserve">
     <value>Maximized</value>
   </data>
+  <data name="Globals_WindowingBehavior.Header" xml:space="preserve">
+    <value>Windowing behavior</value>

```

Comment:
Hmm. Should we workshop the phrasing?

```
New instances of Terminal open in...
... a new window
... the most recently-used window on the same desktop
... the most recently-used window on any desktop
```

How do other applications with automatic glomming phrase this? This seems a little too "technical" for the SUI

Reply:
I couldn't find anyone else with a setting for this. That being said, after the Tabbed Sets fiasco, I'm sure that it's a needed setting, so let's go with the proposal here. I like it.